### PR TITLE
Release1

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

File: dnf-plugins-core.spec [hunk 1]
```diff
@@ -147,6 +147,16 @@ PYTHONPATH=./plugins nosetests-%{python3_version} -s tests/
...
 
 %changelog
+* Thu Feb 09 2017 Jaroslav Mracek <jmracek@redhat.com> 1.0.1-1
+- bump version to 1.0.1 + update release notes (Jaroslav Mracek)
+- setup SideCI to ignore some PEP8 violations (Jaroslav Mracek)
+- spec: define all configs as (noreplace) (Igor Gnatenko)
+- spec: include __pycache__ files (Igor Gnatenko)
+- builddep: print errors from RPM SPEC parser (Petr Spacek)
+
+* Thu Feb 09 2017 Jaroslav Mracek <jmracek@redhat.com> 2.0.1-1
```
Comment (ignatenkobrain) [inline dnf-plugins-core.spec]: no

File: dnf-plugins-core.spec [hunk 2]
```diff
@@ -9,8 +9,8 @@
 %endif
 
 Name:           dnf-plugins-core
-Version:        1.0.0
-Release:        0.rc1.1%{?dist}
+Version:        1.0.1
```
Comment (ignatenkobrain) [inline dnf-plugins-core.spec]: 1.0.0, not 1.0.1

Comment (j-mracek) [inline dnf-plugins-core.spec]: I think that version 1.0.1 is fine

Comment (j-mracek) [inline dnf-plugins-core.spec]: @ignatenkobrain But what I have a problem is to use ```make doc`` for update summaries_cache. Any idea why?

**Comment by j-mracek** [comment]
test this please

**Pull request merged**